druntime/test/betterc/src/test18472.d

```diff
@@ -0,0 +1,37 @@
+/*******************************************/
```

Comment:
This could be a compiler test rather than a druntime one I think

Reply:
Where would that go? I couldn't find

Reply:
compiler/test/compilable or maybe runnable if absolutely necessary

Reply:
Does it run as -betterC explicitely?